dal/reddis.go

```diff
@@ -0,0 +1,55 @@
+package dal
+
+import (
+	"fmt"
+	"github.com/go-redis/redis"
+)
+
+func rClient() *redis.Client {
+	client := redis.NewClient(&redis.Options{
+		Addr: "localhost:6379",
+	})
+	return client
+}
+func ReddisAddRecord(username string, req string){
```

Comment:
no need for the `Redis` prefix, and I think that `action` \ `activity` is better than `Record` WDYT?